cellrank/tl/kernels/_velocity_kernel.py

```diff
@@ -17,23 +17,21 @@
 from cellrank.tl._constants import _DEFAULT_BACKEND, ModeEnum
 from cellrank.ul._parallelize import parallelize
 from cellrank.tl.kernels._utils import (
-    _HAS_JAX,
     prange,
     np_mean,
     _filter_kwargs,
     _random_normal,
     _reconstruct_one,
     _calculate_starts,
     _get_probs_for_zero_vec,
-    _predict_transition_probabilities_jax_H,
-    _predict_transition_probabilities_numpy,
 )
 from cellrank.tl.kernels._base_kernel import (
     _RTOL,
     _LOG_USING_CACHE,
     _ERROR_EMPTY_CACHE_MSG,
     AnnData,
 )
+from cellrank.tl.kernels._velocity_schemes import Scheme, _get_scheme
```

Comment:
`Scheme` is the base class?

Reply:
`Scheme` is the basis for `Cosine`, `Correlation` and `DotProduct`, which also includes the hessian (it has somewhat speciic constructor).

Reply:
okay, thanks.